hack/lib/init.sh

```diff
@@ -11,6 +11,8 @@ set -o pipefail
 
 OS_SCRIPT_START_TIME="$( date +%s )"; export OS_SCRIPT_START_TIME
 
+FLUENTD_WAIT_TIME=120
```

Comment:
The bar is usually _really_ high to add anything to this file. Especially since it's coming from Origin and this will become a long-term carry commit on top. I think it's fine to just use 120 in the areas you need it